task/06-conditions-n-loops-tasks.js

```diff
@@ -251,10 +283,23 @@ function reverseInteger(num) {
  *   4916123456789012 => false
  */
 function isCreditCardNumber(ccn) {
-    throw new Error('Not implemented');
+    let strCCN = ccn.toString();
+    let sum = Number(strCCN[strCCN.length - 1]);
```

Comment:
Don't use `Number` to convert string to number)) Use unary `+` or `parseFloat/parseInt`.